datamade/just-spaces
# Remove passphrase from server GPG key to fix deployments

## Overview

Closes #112.

Remove the passphrase from the server GPG key in order to ensure that CodeDeploy can automatically decrypt application secrets.

## Notes

It had been so long since CodeDeploy ran successfully that the staging server database was incompatible with our migrations. I couldn't figure out how to restore properly from the dump in this repo (the dump was generated on Postgres 10 while the staging server runs 9.6, AFAIK that means they're not compatible) so in order to get CodeDeploy passing again I skipped the restore step and just created an empty database. If you've successfully restored that dump to the staging server, I'd love a quick tutorial on how to do it.

## Testing Instructions

 * Confirm that the manually-triggered CodeDeploy build for this PR ran successfully: https://console.aws.amazon.com/codesuite/codedeploy/deployments/d-HNRJEC4PZ?region=us-east-1
